src/readsbml.jl

```diff
@@ -127,12 +127,14 @@ m = readSBML("my_model.xml", doc -> begin
 end)
 ```
 """
-readSBML(
+function readSBML(
     fn::String,
     sbml_conversion = document -> nothing;
     report_severities = ["Fatal", "Error"],
-)::SBML.Model = _readSBML(:readSBML, fn, sbml_conversion, report_severities)
-
+)::SBML.Model
+    !isfile(fn) && error("$fn is not a file")
```

Comment:
- avoid too much logic (use ||)
- throw the correct kind of exception
- will this work with stuff like named pipes?